Part2-API-Schemas/openapi.yaml

```diff
@@ -35,13 +35,13 @@ components:
     AssetIds:
       name: assetIds
       in: query
-      description: A list of specific Asset identifiers. Each Asset identifier is a base64-url-encoded array of [AssetLinks](#/components/schemas/AssetLink)
+      description: A list of specific Asset identifiers. The value is a base64-url-encoded array of [AssetLinks](#/components/schemas/AssetLink)
```

Comment:
I expected it to be the other way around
`description: every single value within the array of asset identifiers is base64-url-encoded.`

Reply:
I merged your proposal with the latest state.